src/g_shared/a_quake.cpp

```diff
@@ -52,6 +52,9 @@ DEarthquake::DEarthquake (AActor *center, int intensityX, int intensityY, int in
 	m_WaveSpeedX = (float)waveSpeedX;
 	m_WaveSpeedY = (float)waveSpeedY;
 	m_WaveSpeedZ = (float)waveSpeedZ;
+	m_Falloff = falloff << FRACBITS;

```

Comment:
I had the same question, but I never could really understand why Randi set up tremrad like that.

If you check this out:
https://github.com/rheit/zdoom/commit/156d1e61fdf2cef56442be33fd404e174a41d5a0#diff-a5deb3531d905cf2b0c34dfd254b903fR37

You'll see how tremrad is also an int.